### PR TITLE
feat: shrink bundle to only confluent client

### PR DESCRIPTION
**Pull request by aaron-steinfeld**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -35,13 +34,21 @@ repositories {
...
+  extendsFrom(bundled)
+}
+
 dependencies {
   shadow(gradleApi())
-  implementation("com.commercehub.gradle.plugin:gradle-avro-plugin:0.19.0")
+  shadow("com.commercehub.gradle.plugin:gradle-avro-plugin:0.19.0")
   // avro - compiler, tools
-  implementation("org.apache.avro:avro-compiler:1.9.0")
+  shadow("org.apache.avro:avro-compiler:1.9.0")
```
Comment (tim-mwangi) [inline build.gradle.kts]: I've been updating this to 1.9.2

File: build.gradle.kts [hunk 2]
```diff
@@ -35,13 +34,21 @@ repositories {
...
+}
+
+configurations.implementation {
+  extendsFrom(bundled)
+}
+
 dependencies {
   shadow(gradleApi())
-  implementation("com.commercehub.gradle.plugin:gradle-avro-plugin:0.19.0")
+  shadow("com.commercehub.gradle.plugin:gradle-avro-plugin:0.19.0")
```
Comment (tim-mwangi) [inline build.gradle.kts]: jcenter has 0.19.1

File: build.gradle.kts [hunk 3]
```diff
@@ -35,13 +34,21 @@ repositories {
...
   // avro - compiler, tools
-  implementation("org.apache.avro:avro-compiler:1.9.0")
+  shadow("org.apache.avro:avro-compiler:1.9.0")
   // for compatibility checker library
-  implementation("io.confluent:kafka-schema-registry-client:5.4.2")
+  bundled("io.confluent:kafka-schema-registry-client:5.4.2")
```
Comment (tim-mwangi) [inline build.gradle.kts]: 5.5.0 if you dare?

File: build.gradle.kts (hunk 2 above)
Comment (aaron-steinfeld) [reply build.gradle.kts]: sure, when in rome

**Review by tim-mwangi — APPROVED** [review]

**Pull request merged**